ministryofjustice/laa-apply-for-criminal-legal-aid
# Crimap 46 dwp check

## Description of change

Adds DWP benefit checker service and service unreachable page

## Link to relevant ticket

https://dsdmoj.atlassian.net/jira/software/projects/CRIMAP/boards/897?selectedIssue=CRIMAP-46
https://dsdmoj.atlassian.net/jira/software/projects/CRIMAP/boards/897?selectedIssue=CRIMAP-256

## Screenshots of changes (if applicable)

![0 0 0 0_3000_applications_c92e37a6-a17b-458c-9157-d1a2efe43ff4_steps_client_retry_benefit_check](https://user-images.githubusercontent.com/367349/217293384-af05542f-d3da-488b-a86c-87ebc584fcd3.png)

## How to manually test the feature

Get the settings from `config\secrets.yml` and add them to `.env.development.local`. Set `BC_USE_DEV_MOCK` to false.

Use the following credentials to get a positive result

```
last name: Walker'
date of birth: 10/1/1980
national insurance number:  JA293483A
```

Go to `/client/retry_benefit_check` to see the retry page
